Source/UnitTests/Core/DSP/DSPTestText.cpp

```diff
@@ -1,4 +1,10 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
 
+#include "DSPTestText.h"
+
+const char* s_dsp_test_text = R"(
```

Comment:
Just make the size explicit. Your compiler can actually yell at you (with no external facing size, it cannot do that) that way.